### ISSUE TITLE
Add a machine-parsable 'helm version' command

### ISSUE THREAD

**Issue by arschles** [kubernetes/helm#1654]
Right now, `helm version` output looks like this:

```console
ENG000656:service-catalog aaronschlesinger$ helm version
Client: &version.Version{SemVer:"v2.0.1", GitCommit:"dc87ea8dd0111164c720fd13b4eb8a905f97ac62", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.0.1", GitCommit:"dc87ea8dd0111164c720fd13b4eb8a905f97ac62", GitTreeState:"clean"}
```

This is fairly hard to parse in code, so it would be nice to have a `helm` command that produced more easily parsable version output. Shell scripts and similar code that reads STDOUT data would be the primary consumers of this output.

See below for an example (solely for illustrative purposes, not intended to indicate what I think a final design should look like):

```console
ENG000656:service-catalog aaronschlesinger$ helm version --short
client=v2.0.1
server=v2.0.1
```

**Comment by technosophos** [comment]
Personally, I'd like this very much. We followed the kubectl format, and I feel like it's really hard to visually parse.

I think `client=v2.0.0+1e44c98-dirty` would be fine for the more complicated case, too.
